src/Components/Users/Users.tsx

```diff
@@ -127,62 +129,69 @@ export default function Users() {
-              <tr>
-
-                <th scope="col">user Name</th>
-                <th scope="col">Statues</th>
-                <th scope="col">Phone Number</th>
-                <th scope="col">Email</th>
-                <th scope="col">Date Created</th>
-                <th scope="col">Action</th>
-
-              </tr>
-            </thead>
-            <tbody>
-              {userList.map((user) => (
-
-                <tr key={user.id} >
-
-                  <td >{user.userName}</td>
-                  <td >
-                    {user.isActivated ?
-                      (
-                        <div className={`${style.active} ${style.badgeActiveInActive}`}>Active</div>) : (
-                        <div className={`${style.inactive} ${style.badgeActiveInActive}`}>InActive</div>
-                      )}
-                  </td>
-                  <td >{user.phoneNumber}</td>
-                  <td >{user.email}</td>
-                  <td >{new Date(user.group.creationDate).toLocaleDateString()}</td>
-                  <td>
-
-                    <button className={`${style.blockBtn} mx-2`} onClick={() => toggleActivationStatus(user.id, user.isActivated)}>
-                      {user.isActivated ? 'Block' : 'Unblock'}
-                    </button>
-                    <i onClick={() => showViewModel(user.id)}
-                      className="fa-solid fa-eye text-success"></i>
-                  </td>
-                </tr>
 
-              ))}
-            </tbody>
-          </table>
-        </div>
-        :
-        <NoData />
+            <table className="table">
+              <thead className='table-head table-bg'>
+                <tr>
 
-      }
+                  <th scope="col">user Name</th>
+                  <th scope="col">Statues</th>
+                  <th scope="col">Phone Number</th>
+                  <th scope="col">Email</th>
+                  <th scope="col">Date Created</th>
+                  <th scope="col">Action</th>
+
+                </tr>
+              </thead>
+              <tbody>
+                {userList.map((user) => (
+
+                  <tr key={user.id} >
+
+                    <td >{user.userName}</td>
+                    <td >
+                      {user.isActivated ?
+                        (
+                          <div className={`${style.active} ${style.badgeActiveInActive}`}>Active</div>) : (
+                          <div className={`${style.inactive} ${style.badgeActiveInActive}`}>InActive</div>
+                        )}
+                    </td>
+                    <td >{user.phoneNumber}</td>
+                    <td >{user.email}</td>
+                    <td >{new Date(user.group.creationDate).toLocaleDateString()}</td>
+                    <td>
+
+                      <button className={`${style.blockBtn} mx-2`} onClick={() => toggleActivationStatus(user.id, user.isActivated)}>
+                        {user.isActivated ? 'Block' : 'Unblock'}
```

Comment:
here we can say that we have two buttons (texts) for opposite  actions
positive/success: UnBlock
negative/danger: Block

but we are using the red color for both of them, try having a different background color for the UnBlock text such as green.